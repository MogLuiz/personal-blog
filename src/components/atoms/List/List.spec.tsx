import { render, screen } from '@testing-library/react'

import { List } from './List'

describe('Components > Atoms - List', () => {
  it('should be able to render the list correctly', () => {
    const { container } = render(<List>List</List>)
    const listChildren = screen.getByText(/list/i)

    expect(listChildren).toBeVisible()
    expect(container.firstChild).toMatchSnapshot()
  })
})
