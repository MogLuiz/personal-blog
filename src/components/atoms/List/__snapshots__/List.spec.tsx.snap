// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Atoms - List should be able to render the list correctly 1`] = `
<ul
  class="flex justify-between gap-3"
>
  List
</ul>
`;
